com.unity.render-pipelines.core/Editor/CoreEditorUtils.cs

```diff
@@ -273,7 +273,6 @@ public static bool DrawHeaderFoldout(GUIContent title, bool state, bool isBoxed
             var menuRect = new Rect(labelRect.xMax + 3f, labelRect.y + 1f, 16, 16);
 
             // Add context menu for "Additional Properties"
-            Action<Vector2> contextAction = null;
             if (hasMoreOptions != null)
```

Comment:
We knew before that `contextAction` was always null in this code path and therefore we were good to initilize it to the `...OnContextClick..`, but now this is ok?  I mean we are overriding the `Action` even if was not null, so maybe we want to check here if it is null or not.

Reply:
fixed that:) thanks!